src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -126,105 +87,37 @@ public void deleteModificationGroup(UUID groupUuid, boolean errorOnGroupNotFound
         networkModificationRepository.deleteModificationGroup(groupUuid, errorOnGroupNotFound);
     }
 
-    private NetworkModificationException handleException(NetworkModificationException.Type typeIfError, Reporter subReporter, Exception e) {
-        NetworkModificationException networkModificationException;
-        networkModificationException = e instanceof NetworkModificationException ? (NetworkModificationException) e : new NetworkModificationException(typeIfError, e);
-        boolean isApplicationException = PowsyblException.class.isAssignableFrom(e.getClass());
-        if (!isApplicationException && LOGGER.isErrorEnabled()) {
-            LOGGER.error(e.toString(), e);
-        }
-        String errorMessage = isApplicationException ? e.getMessage() : "Technical error: " + e;
-        subReporter.report(Report.builder()
-            .withKey(typeIfError.name())
-            .withDefaultMessage(errorMessage)
-            .withSeverity(TypedValue.ERROR_SEVERITY)
-            .build());
-        return networkModificationException;
-    }
-
-    private ModificationNetworkInfos getNetworkModificationInfos(UUID networkUuid, String variantId) {
+    public NetworkInfos getNetworkInfos(UUID networkUuid, String variantId) {
         Network network;
         try {
             network = networkStoreService.getNetwork(networkUuid);
         } catch (PowsyblException e) {
             throw new NetworkModificationException(NETWORK_NOT_FOUND, networkUuid.toString());
         }
-        boolean applyModifications = true;
+        boolean isVariantExist = true;
```

Comment:
```suggestion
        boolean doesVariantExist = true;
```

Reply:
The getter Lombok adds a prefixe `is` for boolean type

Reply:
So you mean if you call it doesVariantExist, it will be isDoesVariantExist() ?
Then maybe call the variable variantPresent, because isVariantExist is not correct in english.

Reply:
Yes

Reply:
There is still 3 isVariantExist

Reply:
There is still 3 isVariantExist remaining

Reply:
Done